client/src/pages/OfferHelp.js

```diff
@@ -133,6 +133,10 @@ const Step3 = (props) => {
         Question {props.currentStep}/{props.totalSteps}
       </WizardProgress>
       <StepTitle>What is your email address?</StepTitle>
+      <StyledDiv>
+        We respect your privacy. Please read our{" "}
```

Comment:
Also here.

Reply:
Same comment as above ! :) This should be added if we have a space needed before link.